libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -5,6 +5,8 @@
 #include <AP_Logger/AP_Logger.h>
 #include <AP_Landing/AP_Landing.h>
 
+#define MASK_LOG_TECS                   (1<<11)
```

Comment:
Should be passed in the constructor. Or at least leave a big comment in the plane explaining that it is re-defined here.

Reply:
agreed, this mask like this is a total hack.

Reply:
being a poor programmer, I can only go by the patterns established in the code:

 like here: https://github.com/ArduPilot/ardupilot/blob/1c78baac72659a70cbdad055030242e10c9c9194/libraries/AP_Baro/AP_Baro.cpp#L951

and  here https://github.com/ArduPilot/ardupilot/blob/1c78baac72659a70cbdad055030242e10c9c9194/libraries/AP_Generator/AP_Generator_RichenPower.cpp#L312

and here https://github.com/ArduPilot/ardupilot/blob/1c78baac72659a70cbdad055030242e10c9c9194/libraries/AP_Compass/AP_Compass.cpp#L1633

and here https://github.com/ArduPilot/ardupilot/blob/1c78baac72659a70cbdad055030242e10c9c9194/libraries/AP_Generator/AP_Generator_IE_2400.cpp#L188

Reply:
changed